evmcore/dummy_block.go

```diff
@@ -49,7 +49,7 @@ type (
 
 		BaseFee *big.Int
 
-		PrevRandao common.Hash
+		PrevRandao common.Hash // == mixHash/mixDigest
```

Comment:
I am clueless what this added commnet say

Reply:
prevRandao is often saved as a `mixHash` or `mixDigest`. If you have a look at the integration test, you can see I get the random value using `MixDigest()` method 
```
fromLatestBlock := block.MixDigest().Big()
```